packages/trader/src/AppV2/Utils/trade-types-utils.tsx

```diff
@@ -26,3 +29,21 @@ export const AVAILABLE_CONTRACTS = [
     { tradeType: <Localize i18n_default_text='Even/Odd' />, id: CONTRACT_LIST.EVEN_ODD },
     { tradeType: <Localize i18n_default_text='Over/Under' />, id: CONTRACT_LIST.OVER_UNDER },
 ];
+
+export const getTradeTypesList = (contract_types_list: ReturnType<typeof useTraderStore>['contract_types_list']) => {
```

Comment:
this function seems to be missing tests

Reply:
yes, because it's temporary. for temporary Trade types that we had to add. But I ensure you it works well ;)

Reply:
I didn't add the test for this function, because it's temporary function) When Akmal finish trade types, we will apply his logic for getting the list of available trade types and remove this function

Reply:
i was just trying to understand whats happening so i was looking for the tests for input and output to get a better understanding